### DISCUSSION TITLE
ansible-semaphore-9.5.0 2.10 - bad boolean environment value '0

### DISCUSSION THREAD

**Discussion by monique77** [semaphoreui/semaphore#2109] [General]
Hello, Guys

I would like your help, if possible. So, I'm trying to configure a repository in my github through of ansible semaphore, but it has a error bellow;

I configured the key private, and I putted the public key in my github. In my environment has {} but I tried to put { "environment": "localhost"
}
But I have the same error. In the documentation I didn't see this specific error.
I will appreciate your help if possible you help me.
Tks so much

Versions
kubernetes v1.30.1
ansible-semaphore-9.5.0 2.10
Helm v3.15.1

1:55:09 PM
Cloning into 'repository_4_7'...
1:55:09 PM
fatal: bad boolean environment value '0
1:55:09 PM
' for 'GIT_TERMINAL_PROMPT'
1:55:09 PM
Failed updating repository: exit status 128
1:55:09 PM
Running playbook failed: exit status 128

**Comment by BlockyBlockling** [comment]
Maybe this can help?
https://github.com/semaphoreui/semaphore/discussions/2015

**Comment by rome-legacy** [comment]
I had the same issue.
I'm using gitlab free tier with an access token.
I have updated my secret (seemingly token was properly saved or pasted).
Afterwards it was ok.

**Comment by jeremy-bepoix** [comment]
Hi, 
I encountered the same problem after adding a submodule on my repository.
The solution was to change the full url with relative url on my `.gitmodules`

credit : 
- https://www.damirscorner.com/blog/posts/20210423-ChangingUrlsOfGitSubmodules.html
- https://support.atlassian.com/bitbucket-data-center/kb/cloning-submodule-fails-from-mirror

regards
